examples/cell_tracking.py

```diff
@@ -0,0 +1,81 @@
+"""
+Cell tracking example
+=====================
+
+This example demonstrates how the track layer is used
+for visualizing cell tracking data, by displaying 2D + time dataset
+of cells with colored properties for track-id.
+
+Thanks to Dr. Alessia Ruggieri and Philipp Klein, Centre for Integrative Infectious
+Disease Research (CIID), University Hospital Heidelberg, Germany for the data.
+You can find the data on: https://doi.org/10.5281/zenodo.15597019
+
+.. tags:: visualization-advanced, colored 2D tracks
+"""
+import zipfile
+from pathlib import Path
+
+import numpy as np
+import pooch
+import tifffile
+from trackastra.model import Trackastra
+from trackastra.tracking import ctc_to_napari_tracks, graph_to_ctc
+
+import napari
+
+# Create temporary directory
+tmp_dir = Path(pooch.os_cache('napari-cell-tracking-example'))
+tmp_dir.mkdir(parents=True, exist_ok=True)
+
+# Extract silver truth data for trackastra
+st_url = "https://zenodo.org/records/15852284/files/masks_pred.npz?download=1"
+st_path = pooch.retrieve(
+    url=st_url,
+    fname="masks_pred.npz",
+    known_hash=None,
+    path=pooch.os_cache("napari cell tracking example")
+)
```

Comment:
I can confirm you can do the unzipping in the retrieve stage with `processor=pooch.Unzip()`. So you can clean up the code additionally by switching to this logic, which will return a list of the files.
https://www.fatiando.org/pooch/latest/api/generated/pooch.Unzip.html#pooch.Unzip

```suggestion
st_path = pooch.retrieve(
    url=st_url,
    fname="masks_pred.npz",
    known_hash=None,
    path=pooch.os_cache("napari cell tracking example"),
    processor=pooch.Unzip(),
    progressbar=True,
)
```

Reply:
Thanks Tim! The extraction is actually for the tif files and not masks. It's cleaner, now down to 75 lines and I like it. New commit is incoming.